formspree/forms/helpers.py

```diff
@@ -8,7 +8,7 @@
 
 from formspree import settings
 
-HASH = lambda x, y: hashlib.md5(x+y+settings.NONCE_SECRET).hexdigest()
+HASH = lambda x, y: hashlib.md5(x.encode('utf-8')+y.encode('utf-8')+settings.NONCE_SECRET).hexdigest()
```

Comment:
This is the line which should be looked more carefully, is it ok? It looks ok to me, but I'm biased.